### ISSUE TITLE
Where are the OCaml communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/ocaml#256]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is OCaml supported by one or more large organizations?
* Does OCaml have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about OCaml? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for OCaml? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in OCaml? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in OCaml?
* Are there popular newsletters for OCaml?
* Is OCaml taught at programming bootcamps? (If so, what are some examples?)
* Is OCaml taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about OCaml?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by stevejb71** [comment]
> Is OCaml supported by one or more large organizations?
INRIA, a French public research body developer Ocaml
Jane Street have developed Core, an alternative standard library for the language. They look to be heavily invested in it.
Facebook use Ocaml internally, and have written ReasonML which is a syntax overlay for it, compiling to JavaScript.

> Do you know of specific communities (online or offline) that are enthusiastic about OCaml? (Chat communities, forums, meetups, student clubs, etc)
There are is a google group for OCaml Core (Jane street libary). I've asked questions on Core and found them to be helpful. 
There are meetup groups for Ocaml - https://www.meetup.com/topics/ocaml-programming/

https://groups.google.com/forum/#!forum/ocaml-core

> Are there popular newsletters for OCaml?
Ocaml weekly news: http://ocaml.org/community/cwn/

> Is OCaml taught at universities? (If so, what are some examples?)
Yes - e.g. 
Cornell: https://www.cs.cornell.edu/courses/cs3110/2012sp/courseinfo.php
Princeton: http://www.cs.princeton.edu/~dpw/courses/cos326-12/info.php

**Comment by kytrinyx** [comment]
This is great, thanks @stevejb71!
